frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -157,6 +185,37 @@ export default class QualityNominationList extends Vue {
     author_username: T.wordsAuthor,
   };
 
+  get orderedNominations(): types.NominationListItem[] {
+    let order = this.sortOrder === omegaup.SortOrder.Ascending ? -1 : 1;
+
+    switch (this.columnName) {
+      case 'title':
+        return this.nominations.sort((a, b) =>
+          a.problem.title < b.problem.title
+            ? order
+            : b.problem.title < a.problem.title
+            ? -1 * order
+            : 0,
+        );
+      case 'time':
+        return this.nominations.sort((a, b) =>
+          a.time.getTime() < b.time.getTime()
+            ? order
+            : b.time.getTime() < a.time.getTime()
+            ? -1 * order
+            : 0,
```

Comment:
```suggestion
          order * (a.time.getTime() - b.time.getTime(),
```

esto es más conciso.

Reply:
De hecho mi solución fue más concisa que que la que aplicaron en componentes de otras interfaces con "if", pero en efecto, observando nuevamente la operación, se puede reducir aún más como tú propones, así que copiaré la línea de código tal cual, gracias por la observación.